docs/advanced/transformers/index.md

```diff
@@ -7,10 +7,9 @@ redirect_from:
  - /docs/usage/transformers/
 ---
 
-A transformer is a module with a specific goal that is ran against your code to transform it. For example,
-the `es6.arrowFunctions` transformer has the very specific goal of transforming [ES6 Arrow Functions](https://babeljs.io/docs/learn-es2015/#arrows)
-to the equivalent ES3. This allows transformers to be completely unaware of the other transformations happening
-so that you can easily chain them together.
+A transformer is a module that processes your code to perform a specific, focused transformation such that transformers can be completely unaware of each other and therefore easily chained together. For example, the `es6.arrowFunctions` transformer has the very specific goal of transforming [ES6 Arrow Functions](https://babeljs.io/docs/learn-es2015/#arrows) to the equivalent ES3.
```

Comment:
> `specific, focused` sounds kind of redundant?

Yes. I'd be ok with just "focused". I think "specific goal" was redundant to begin with -- obviously it's not going to have a vague unspecified goal of transforming the code :)

> I think we're just trying to say it only does one thing and does it independently to any other transformer?

That sounds too vague, or perhaps too inaccurately specific, to me. What does "one thing" consist of? This is what I had in mind when I chose the word "focused".

> I might use words like independent or isolated as well?

I'm not married to the language I proposed here. Feel free to post the full blurb you have in mind in a comment here, or perhaps better yet grab my branch and add a commit.

Thanks!